src/content/profile-tree.js

```diff
@@ -109,16 +116,20 @@ class ProfileTree {
     if (node === undefined) {
       const funcIndex = this._funcStackTable.func[funcStackIndex];
       const funcName = this._stringTable.getString(this._funcTable.name[funcIndex]);
+      const resourceIndex = this._funcTable.resource[funcIndex];
+      const resourceType = this._resourceTable.type[resourceIndex];
       const isJS = this._funcTable.isJS[funcIndex];
+      const libName = this._getOriginAnnotation(funcIndex);
```

Comment:
I don't understand everything in _getOriginAnnotation, and so I'm not sure what I can get as return value. Do we always get a URL for webhost types ?